sdk-extensions/autoconfigure/src/test/java/io/opentelemetry/sdk/autoconfigure/DefaultConfigPropertiesTest.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.autoconfigure;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+import io.opentelemetry.sdk.autoconfigure.spi.ConfigProperties;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+
+class DefaultConfigPropertiesTest {
+
+  @Test
+  void resourceAttributesMerged() {
+    Map<String, String> environment = new HashMap<>();
+    Map<String, String> system = new HashMap<>();
+
+    environment.put(
+        "OTEL_RESOURCE_ATTRIBUTES", "deployment.environment=env,service.name=env.service");
+    system.put("otel.resource.attributes", "deployment.environment=sys");
```

Comment:
What if you set the value to `deployment.environment=`? Would it clear the key in the final map?

Reply:
I would expect so, yeah. I should probably add test coverage.

Reply:
ok, test added.